jazz-ui/src/app/pages/admin-dashboard/admin-dashboard.component.html

```diff
@@ -0,0 +1,36 @@
+<jazz-mob-header (addClick)='showService(true)' [icon]="'add'" [headText]="'Services'"
+                 [status]="'none'"></jazz-mob-header>
+<section class="table-wrap">
+  <div class="page-title-wrap hide-for-mobile">
+    <h1 class="header1 bold-title">Admin</h1>
+  </div>
+  <section class="json-viewer-container">
+    <div class="section-header">
+      <span *ngIf='!isCollapsed' class="icon-Collapse-All expandbtn" (click)='toggleCollapse(true)'></span>
+      <span *ngIf='isCollapsed' class="icon-Expand-All expandbtn" (click)='toggleCollapse(false)'></span>
+    </div>
+
+    <json-viewer #jsonRoot
+                 *ngIf="state === 'resolved'"
+                 [json]="adminData"></json-viewer>
+    <div class="section-centered"
+      *ngIf="state === 'loading'">
+      <div class="loading-circle"></div>
+    </div>
+
+    <div  *ngIf="state === 'error'" class="section-centered">
+      <div class="text-center">
+        <div class="icon-icon-server-error">
+          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span>
+        </div>
+        <div class='oops'>Oops!</div><br>
+        <div>Something went wrong while fetching your data<br><a (click)="ngOnInit()">REFRESH</a> to give it another try.</div>
```

Comment:
It seems we are writing text in English within the component itself. Have we thought about keeping them in a config file. Maybe it could help with localization later. What do you think?